plugins/redfish/fu-plugin-redfish.c

```diff
@@ -119,6 +119,7 @@ fu_plugin_init (FuPlugin *plugin)
 {
 	FuPluginData *data = fu_plugin_alloc_data (plugin, sizeof (FuPluginData));
 	data->client = fu_redfish_client_new ();
+	fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_BETTER_THAN, "uefi");
```

Comment:
Is this a likely occurrence between these two plugins? I would think the ESRT guid isn't likely to be the same as a redfish GUID.

Or is this just to reflect the scenario you were mentioning in tests?

Reply:
Well, it's not required at all; it was just a real world example that seemed plausible. I can drop this if you'd prefer.

Reply:
Can @lcp maybe comment on this to confirm if there is any relation in the hpe implementation?

If not I think it's better to drop it until there is a similar type of relationship we know of in the wild.

Reply:
The HPE Gen10 I'm using doesn't have ESRT, and there is no devices from UEFI plugin. So I guess this change won't affect HPE machines, at least for now.

Reply:
OK thanks for confirming.